build.sbt

```diff
@@ -89,6 +89,10 @@ val unitTestingStack = Seq(scalatest)
 val embeddedPostgres = "com.opentable.components" % "otj-pg-embedded" % "0.13.3" % Test
 val dbTestingStack = Seq(embeddedPostgres)
 
+val catsEffectStack = Seq(
+  "org.typelevel" %% "cats-effect-laws" % "2.2.0" % Test
```

Comment:
do we need this dep?

Reply:
yeah, without this I couldn't use   implicit lazy val ec: TestContext = TestContext.apply()
to control time in tests
https://blog.softwaremill.com/time-traveling-in-tests-with-cats-effect-b22084f6a89

Reply:
ah sure, thanks :)